src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -1190,6 +1192,33 @@ public Reader renderMarkdown(String text) throws IOException {
                 "UTF-8");
     }
 
+    /**
+     * Gets an {@link ObjectWriter}. Do not use this method.
+     *
+     * @return an {@link ObjectWriter} instance that can be further configured.
+     * @deprecated Provided for backward compatibility with projects that did their own jackson mapping of this
+     *             project's data objects, such as Jenkins Blue Ocean.
```

Comment:
It might be helpful to mention or link to the API you are supposed to use instead of these methods.

Reply:
There is none.  No one should use these methods.  No one should have used them to begin with.  The scenario is unsupported and should never have been done.

Reply:
I will add a test to make sure these methods continue to work for now.  I'm not sure how I could stop someone from doing the same a Blue Ocean did, but if it were any other client aside from a Jenkins plugin I would not be making this change.

Reply:
> There is none. No one should use these methods. No one should have done anything like this to begin with The scenario is completely unsupported.

Yeah, but I mean that it's helpful to point users in the right direction so they understand what scenario _is_ supported. Maybe this approach is so wrong that all you can say is "See the documentation at XYZ for how this library is intended to be used", but some kind of reference to the correct approach seems helpful in case someone sees this method and thinks "hey, this looks like it does what I want".